### DISCUSSION TITLE
element-plus组件穿透footer问题

### DISCUSSION THREAD

**Discussion by schunlee** [element-plus/element-plus#1891] [Q&A]
不好意识，我是个前端新手，这里遇到个问题，随着页面而滚动的footer被内容中的组件穿透了，请问应该如何解决这个问题。

我的footer css: 

       #footer {
          position: fixed;
          bottom: 0;

遇到的页面问题:
[https://i.imgur.com/udA9Qbp.png](https://i.imgur.com/udA9Qbp.png)
[https://i.imgur.com/gsb18nH.png](https://i.imgur.com/gsb18nH.png)

**Comment by jw-foss** [comment]
Sorry for the late update, you can set the `z-index` value for your footer element, so that you won't get this issue again. You can starting by setting the z-index of your footer to a relatively big value and try reduce it half way down to get a reasonable value (because it is not suggested to set z-index to a big number which could cause the layout disordering).

**Reply by jw-foss** [reply]
Hope my answer could solve your problem. Next time please open an issue to track the problems you encountered. Discussion is discussion which should be used as discussion instead of issue.
